secrets.test.js

```diff
@@ -47,6 +45,6 @@ test('export dynamic secrets', async () => {
         'token': 'akeyless-token',
         'names': ['/path/to/static/secret'],
     });
-    expect(core.setSecret).toHaveBeenCalledWith('sup', 'super secret');
```

Comment:
Same with the above - let's test that both setSecret and setOutput are both being properly called.